test/unit/jobs/cleanup-test.js

```diff
@@ -103,6 +110,39 @@ describe("Delete unused files", () => {
     }, 200);
   });
 
+  it("should delete financial data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
+      cleanupJob.financial.findByKey("risefinancial_realtime_HXXHCTR5AQFQ_-Kn5MaUE2cpYN-FCQSOU_1M_", (err, foundFinancial) => {
+        expect(Object.keys(foundFinancial.data).length).to.equal(0);
+
+        done();
+      })
+    }, 200);
+  });
+
+  it("should delete spreadsheet data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
```

Comment:
@tejohnso I mean how can I resolve to a single promise if all the tasks on the job runs async? I would have to wait for all of them to resolve, right? I agree that using timeout is not the best just don't know a better alternative. cheers